tests/phpunit/tests/pluggable/wpMail.php

```diff
@@ -477,9 +477,9 @@ public function test_wp_mail_sends_attachments_with_original_name() {
 
 		$attachments = $mailer->getAttachments();
 
-		$this->assertTrue( $mailer->attachmentExists() );
-		$this->assertSame( $attachments[0][1], $attachments[0][2] );
-		$this->assertSame( $attachments[1][1], $attachments[1][2] );
+		$this->assertTrue( $mailer->attachmentExists(), 'There are no attachments.' );
+		$this->assertSame( $attachments[0][1], $attachments[0][2], 'The attachment name did not match.' );
+		$this->assertSame( $attachments[1][1], $attachments[1][2], 'The attachment name did not match.' );
```

Comment:
```suggestion
		$this->assertSame( $attachments[0][1], $attachments[0][2], 'The first attachment name did not match.' );
		$this->assertSame( $attachments[1][1], $attachments[1][2], 'The second attachment name did not match.' );
```
Otherwise, the same message will show and it won't be clear which assertion it refers to.

Reply:
Message updated. Thanks!